packages/marketplace/src/components/pages/client-app-detail/client-app-detail.tsx

```diff
@@ -24,7 +24,12 @@ export const handleInstallAppButtonClick = (setIsVisibleInstallConfirmation: (is
   }
 }
 
-export const renderAppHeaderButtonGroup = (id: string, installedOn: string, onInstallConfirmationModal: () => void) => {
+export const renderAppHeaderButtonGroup = (
+  id: string,
+  installedOn: string,
+  onInstallConfirmationModal: () => void,
+  isInstallBtnDisabled: boolean,
```

Comment:
It's not disabled though, it's hidden right? Can we change this param so it's clearer what is going on.